docs/utd_juno.md

```diff
@@ -0,0 +1,39 @@
+# nf-core/configs: UTD Juno Configuration
+
+TBD: All nf-core pipelines have been successfully configured for use on the [Juno HPC cluster](https://docs.circ.utdallas.edu/user-guide/systems/juno.html) at [The Univeristy of Texas at Dallas](https://www.utdallas.edu/).
+
+To use, run the pipeline with `-profile utd_juno`. This will download and launch the [`utd_juno.config`](../conf/utd_juno.config) which has been pre-configured with a setup suitable for Juno.
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
+
+> [!NOTE]
+> You will need an account to use the HPC cluster on Ganymede in order to run the pipeline.
```

Comment:
Personal opinion; maybe it would make sense to have a single profile to for UTD systems to make life easier for users if the systems are aligned enough?

Reply:
I am not sure about the entire situation but we would certainly need an escape hatch, it doesn't seem like in the near future access to all clusters would be unified.

Or are you talking about we try to detect which system we are running on, and then select the queue?

Reply:
There may be different opinions and I'm not witholding approval because of this, but to me it typically makes a lot more sense to have a profile for the institution/site/department/provider rather than any number of different profiles for different clusters.

Reply:
@pontus due to profile/config inheritance issues with DSL2, it was recommended to us (somewhere, I can't find it now) that it's better to have singular config files per cluster rather than 'sub profiles'.

So indeed `utd_juno` and then in the future `utd_ganymede` etc, is valid.

Note it also makes it easier to deprecate older clusters etc.

Reply:
I'm not sure what issues with DSL2 that would have been, but there's definitely some things related to the upcoming strict syntax that makes it harder (but to me, not to a degree that I think the added work for configuration maintenance outweighs the burden of users).

But I can say I agree this is something one can hav different opinions about :)

To me, it seems having separate profiles make it harder to deprecate old clusters, rather than just changes in a config file and possibly docs, there's more that needs to be pulled down.